equinity-historical-data/src/main/java/uk/gov/justice/laa/crime/equinity/historicaldata/model/crm14/Crm14Model.java

```diff
@@ -7,6 +7,7 @@
 import uk.gov.justice.laa.crime.equinity.historicaldata.model.CrmEvidenceFileModel;
 import uk.gov.justice.laa.crime.equinity.historicaldata.model.CrmEvidenceFilesModel;
 import uk.gov.justice.laa.crime.equinity.historicaldata.model.CrmFormModelInterface;
+import uk.gov.justice.laa.crime.equinity.historicaldata.model.data.Crm14AttachmentModel;
```

Comment:
We should consider moving function `public void addProcessedAttachmentsToEvidence(...)` to class `CrmEvidenceFilesModel` which would eliminate the extra dependency here

Reply:
shall we do it in this PR ?

Reply:
nope

Reply:
added a TODO comment to check on the future